README.md

```diff
@@ -64,19 +66,20 @@ Create an `Okta.plist` file in your application's bundle with the following fiel
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>issuer</key>
-    <string>https://{yourOktaDomain}.com/oauth2/default</string>
-    <key>clientId</key>
-    <string>{clientIdValue}</string>
-    <key>redirectUri</key>
-    <string>{redirectUrlValue}</string>
-    <key>scopes</key>
-    <string>openid profile offline_access</string>
+<key>issuer</key>
+<string>https://{yourOktaDomain}.com/oauth2/default</string>
+<key>clientId</key>
+<string>{clientIdValue}</string>
+<key>redirectUri</key>
+<string>{redirectUrlValue}</string>
+<key>scopes</key>
```

Comment:
Since we callout below "you must specify the `logoutRedirectUri`, lets add an example to this plist with a value:

```diff
<key>redirectUri</key>
<string>{redirectUriValue}</string>
+<key>logoutRedirectUri</key>
+<string>{logoutRedirectUriValue}</string>
```

Reply:
Done.